### PR TITLE
Pi5 otp doc update

### PR DESCRIPTION
**Pull request by peterharperuk**
Add some details about 2712 OTP rows

### CODE REVIEW

**Review by aallan — APPROVED** [review]

**Comment by aallan** [comment]
@nathan-contino Just pointing you at this for the Raspberry Pi 5 Datasheet.

**Comment by peterharperuk** [comment]
@timg236 Could you check this look ok?

File: documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc [hunk 1]
```diff
@@ -1,16 +1,19 @@
 == OTP Register and Bit Definitions
 
-All SoCs used by the Raspberry Pi range have a inbuilt One-Time Programmable (OTP) memory block.
+All SoCs used by the Raspberry Pi range have a inbuilt One-Time Programmable (OTP) memory block. A few locations have factory-programmed data.
 
-It is 66 32-bit values long, although only a few locations have factory-programmed data.
+OTP memory size:
+
+* 2711: 66 32-bit values
```
Comment (lurch) [inline documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc:7]: Does "2711" here actually mean "BCM2835, BCM2836, BMC2837 & BCM2711" ?

Comment (peterharperuk) [reply documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc:7]: Good point - I actually mean "not 2712"

**Comment by peterharperuk** [comment]
Just force pushed to remove some of the OTP rows mentioned in the first commit

File: documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc [hunk 2]
```diff
@@ -63,3 +66,57 @@ This is available via device-tree in `/proc/device-tree/chosen/rpi-boardrev-ext`
...
+=== 2712 - OTP Registers
+
+This list contains the publicly available information on the registers. If a register or bit is not defined here, then it is not public.
+
+22 -- bootmode register +
+
+* Bit 0: Disable parity checking
+* Bit 1: Boot from SD card
+* Bits 2-4: Booting from SPI EEPROM (and which GPIOs)
+* Bit 5: Enable USB device booting
```
Comment (timg236) [inline documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc:79]: bit 5 is redundant - remove

File: documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc [hunk 3]
```diff
@@ -63,3 +66,57 @@ This is available via device-tree in `/proc/device-tree/chosen/rpi-boardrev-ext`
...
 
 ^1^Also contains bits to disable overvoltage, OTP programming, and OTP reading.
+
+=== 2712 - OTP Registers
+
+This list contains the publicly available information on the registers. If a register or bit is not defined here, then it is not public.
+
+22 -- bootmode register +
+
+* Bit 0: Disable parity checking
```
Comment (timg236) [inline documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc:76]: I think bit 0 was for debug - remove

**Review by timg236 — COMMENTED** [review]
Looks good

File: documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc (hunk 1 above)
Comment (peterharperuk) [reply documentation/asciidoc/computers/raspberry-pi/otp-bits.adoc]: I've pushed a commit to change these references to "non-BCM2712 devices"

**Pull request merged**